.gitignore

```diff
@@ -99,4 +100,4 @@ package-lock.json
 *.css.map
 *.map
 style.css
-woocommerce.css
```

Comment:
Not sure why the browser editor added this. May be an OS line break related issue.